src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -757,12 +762,25 @@ static function (ChildDefinition $definition, AsPeriodicTask|AsCronTask $attribu
                 }
             );
         }
-        $container->registerAttributeForAutoconfiguration(JsonStreamable::class, static function (ChildDefinition $definition, JsonStreamable $attribute): void {
-            $definition->addTag('json_streamer.streamable', [
+
+        $container->registerForAutoconfiguration(CompilerPassInterface::class)
+            ->addExcludeTag('container.excluded.compiler_pass');
+        $container->registerForAutoconfiguration(KernelInterface::class)
```

Comment:
Can the kernel still be used as a command? https://github.com/symfony/symfony/pull/59979

Reply:
Yes